shema.sql

```diff
@@ -20,3 +20,18 @@ CREATE TABLE LABEL (
   book_id INT,
   FOREIGN KEY (book_id) REFERENCES item(id) ON DELETE CASCADE
 );
+
+CREATE TABLE GAME (
+  id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  multiplayer VARCHAR(50) NOT NULL,
+  last_player_at VARCHAR(50) NOT NULL,
+  FOREIGN KEY (id) REFERENCES item(id) ON DELETE CASCADE
+);
+
+CREATE TABLE AUTHOR (
+  id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  first_name VARCHAR(50) NOT NULL,
+  last_name VARCHAR(50) NOT NULL,
+  game_id INT,
```

Comment:
- [ ] I think it would be better to use `author_id` instead of `game_id` so that we adhere to our [UML](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/images/catalog_of_my_things.png) diagram🥱🥱👌